lib/origami-build-tools-cli.js

```diff
@@ -96,6 +96,8 @@ if (tasks.isValid(argument)) {
 				taskResult
 					.on('error', reportTaskError)
 					.on('end', reportFinished);
+				// Make sure it reaches 'end'
+				taskResult.resume();
```

Comment:
Ok, sure, but in this instance, takResult could define `on` but not `resume` and cause a 'fatal' error, depends on the implementation of the stream being returned.